build.gradle

```diff
@@ -262,6 +262,7 @@ configurations {
             force "io.netty:netty-handler:${versions.netty}"
             force "io.netty:netty-transport:${versions.netty}"
             force "io.netty:netty-transport-native-unix-common:${versions.netty}"
+            force "org.apache.bcel:bcel:6.6.0" // Fixes `issues/2248`. This dependency comes transitively via spotbugs
```

Comment:
Can we remove this comment, having this information in the commit message should be enough

Reply:
I kept this here as a reminder to remove it once spotbugs releases a patch with this fix. Let me update the comment to say as such